modules/proc_deploy-quay-openshift-operator.adoc

```diff
@@ -1120,32 +1300,215 @@ run the operator locally:
 $ operator-sdk up local --namespace=quay-enterprise
 ```
 
-= Upgrading {productname} and Clair
+= Upgrading {productname} 
+The Quay Operator v3.3.0 has many changes from v1.0.2. The most notable which
+affects the upgrade process is the backwards-incompatible change to the CRD.
+Ultimately, the CR (Custom Resource) used to deploy {productname} using the operator
+may have to be modified accordingly.
+
+== Upgrade Prerequisites
+Ensure that your deployment is using a supported persistance layer and
+database. A production {productname} deployment run by the Operator should *not* be
+relying on the Postgres instance or a OpenShift volume that has
+been created by the Operator.
+
+If you are using a Postgres instance or OpenShift volume that was created
+by the Operator, the upgrade path is not suported as the removal of the old
+Operator will cascade the deletion of your database and volume. It may be
+possible to manually migrate your data to supported storage mechanisms but
+this is not within the scope of the typical, or supported, upgrade path.
+
+Please read through the entire guide before following any steps as this upgrade
+path is potentially destructive and there is no guaranteed roll-back mechanism.
+
+== Upgrade Process Summary
+
+Here are the basic steps for upgrading the {productname} cluster
+you originally deployed from the v1.0.2 Quay Setup Operator to
+the v3.3 Quay Operator:
+
+. Document all configuration related to your current deployment.
+. Copy your CR and modify any configuration values as needed.
+. Remove your current deployment using `oc delete -f deployment.yaml`
+. Ensure that only one quay pod will be started, as this Pod will perform any
+database migrations needed before scaling up the entire cluster.
```

Comment:
Is it available for moving this part to recreate quay deployment

Reply:
Oh, I see. This means scale down Quay pod replicas to one

Reply:
That's how I understood it.